src/styles/main.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc((#{$day-size} * 7 + (6 * #{$gap})) + #{$padding} * 2);
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
```

Comment:
The color `#eee` should be moved to a variable for better maintainability and to follow the DRY (Don't Repeat Yourself) principle. This allows you to easily update the color in one place if it is used in multiple places.